samples/batch/csharp/batchclient/program.cs

```diff
@@ -11,34 +11,51 @@ namespace BatchClient
     using System.Threading.Tasks;
     using Newtonsoft.Json;
 
-    class Program
+    public class Program
     {
         // Replace with your subscription key
-        private const string SubscriptionKey = "YourSubscriptionKey";
+        public const string SubscriptionKey = "YourSubscriptionKey";
 
         // Update with your service region
-        private const string Region = "YourServiceRegion";
+        public const string Region = "YourServiceRegion";
+
+        // replace with your app service name (check publish on webhook receiver project)
+        public const string WebHookAppServiceNme = "YouAppServiceName";
+
+        // replace with a secure secret (used for hashing)
+        public const string WebHookSecret = "somethingverysecretisbesthere";
 
         // recordings and locale
         private const string Locale = "en-US";
         private static Uri RecordingsBlobUri = new Uri("<SAS URI pointing to an audio file stored in Azure Blob Storage>");
         //private static Uri ContentAzureBlobContainer = new Uri("<SAS URI pointing to an container in Azure Blob Storage>");
+        private static Uri WebHookCallbackUrl = new Uri($"https://{WebHookAppServiceNme}.azurewebsites.net/api/callback");
 
         // For use of custom trained model:
         private static EntityReference CustomModel = null;
         //private static EntityReference CustomModel =
         //    new EntityReference { Self = new Uri($"https://{Region}.api.cognitive.microsoft.com/speechtotext/v3.0/models/<id of custom model>")};
         private const string DisplayName = "Simple transcription";
 
-#pragma warning disable UseAsyncSuffix // Use Async suffix
-        static async Task Main(string[] args)
-#pragma warning restore UseAsyncSuffix // Use Async suffix
+
+        static void Main(string[] args)
         {
-            Console.WriteLine("Starting transcriptions client...");
+            RunAsync().Wait();
+        }
 
+        private static async Task RunAsync()
+        {
             // create the client object and authenticate
-            var client = BatchClient.CreateApiV3Client(SubscriptionKey, $"{Region}.api.cognitive.microsoft.com");
+            using (var client = BatchClient.CreateApiV3Client(SubscriptionKey, $"{Region}.api.cognitive.microsoft.com"))
+            {
+                // await SetupWebHookAsync(client).ConfigureAwait(false);
```

Comment:
Could you please put a comment why this line is commented out, and when it should be included?